### PR TITLE
community/vdr: move from testing

### PR DESCRIPTION
**Pull request by kr0k0**
update femon and vnsiserver plugin
add epgsearch, dvbapi, svdrpservice and vdrmanager plugin
re-enable skincurses plugin
add init script
add configs for vdr and all plugins
add vdr user and group on install and remove them on uninstall
fix build with debug symbols (include-missing-limits.patch)
fix streamdev Makefile
apply available upstream patches (at the moment 33)

### CODE REVIEW

File: community/vdr/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,186 @@
+# Contributor: Stefan Reiff <kroko87@hotmail.com>
+# Contributor: Carlo Landmeter <clandmeter@gmail.com>
+# Maintainer: Taner Tas <taner76@gmail.com>
+pkgname=vdr
+pkgver=2.4.0
+pkgrel=1
+pkgdesc="Video Disk Recorder"
+url="http://www.tvdr.de/"
+arch="all"
+license="GPL-2.0"
```
Comment (maxice8) [inline community/vdr/APKBUILD]: needs spdx identifier

File: community/vdr/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+depends=""
+depends_dev="libexecinfo-dev fontconfig-dev freetype-dev gettext-dev libjpeg-turbo-dev fribidi-dev
+	libcap-dev alsa-lib-dev libvdpau-dev libva-dev libx11-dev xcb-util-dev xcb-util-wm-dev
+	ffmpeg-dev libcdio-dev mpv"
```
Comment (maxice8) [inline community/vdr/APKBUILD]: shouldn't it be mpv-dev ?

File: community/vdr/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+# Contributor: Carlo Landmeter <clandmeter@gmail.com>
+# Maintainer: Taner Tas <taner76@gmail.com>
+pkgname=vdr
+pkgver=2.4.0
+pkgrel=1
+pkgdesc="Video Disk Recorder"
+url="http://www.tvdr.de/"
+arch="all"
+license="GPL-2.0"
+depends=""
```
Comment (maxice8) [inline community/vdr/APKBUILD]: empty variables can be removed for brevity

**Comment by kr0k0** [comment]
maxice8: Thanks for hints!

File: community/vdr/APKBUILD (hunk 2 above)
Comment (kr0k0) [reply community/vdr/APKBUILD]: mpv-dev isn't available, but package mpv contains the headers

File: community/vdr/vdr.post-deinstall [hunk 4]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+delgroup vdr video 2>/dev/null
+deluser vdr 2>/dev/null
```
Comment (Ikke) [inline community/vdr/vdr.post-deinstall]: You should not remove users from the system in install scripts, you can only add them.

**Comment by kr0k0** [comment]
Ikke: Thanks for review!

**Comment by Ikke** [comment]
This has been merged as f4e6ce7, 4796564. Closing PR.

**Pull request closed without merging**